src/lib.rs

```diff
@@ -11,9 +11,9 @@ pub struct Btc {
 }
 
 pub struct BtcParams {
```

Comment:
Is the struct needed? I think we could away with three parameter &str and it'd even be more efficient

Reply:
Not needed, but looked convenient. `build` has 3 params that are the same type, looked easy to misplace them.